integreat_compass/cms/models/interactions/comment.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ..abstract_base_model import AbstractBaseModel
+from ..offers.offer_version import OfferVersion
+from ..users.user import User
+
+
+class Comment(AbstractBaseModel):
+    """
+    Data model representing a Comment.
+    """
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    offer_version = models.ForeignKey(OfferVersion, on_delete=models.CASCADE)
+    date = models.DateTimeField(auto_now_add=True)
+    comment = models.CharField(
+        max_length=255,
+        blank=False,
+        null=False,
+        verbose_name=_("comment"),
+        help_text=_("Additional context for the given rating"),
+    )
+    rating = models.PositiveSmallIntegerField(
+        choices=((i, i) for i in range(1, 6)),
```

Comment:
If the rating has to be between 1 and 5, I think validators make more sense than choices.
```suggestion
        validators=[
            MinValueValidator(1),
            MaxValueValidator(5),
        ]
```
(requires `from django.core.validators import MaxValueValidator, MinValueValidator`)